aten/src/ATen/native/quantized/cpu/qembeddingbag.cpp

```diff
@@ -141,9 +141,10 @@ at::Tensor& embedding_lookup_fallback_impl(
 }
 
 template <typename IndexType, typename OffsetType>
-at::Tensor& embedding_bag_4bit_impl(
+at::Tensor& embedding_bag_nbit_impl(
     at::Tensor& output,
     const at::Tensor& weight,
+    int bit_width,
```

Comment:
```suggestion
    const int bit_width,
```

make this arg `const`